src/serve/hosting.ts

```diff
@@ -52,13 +52,13 @@ function startServer(options: any, config: any, port: number, init: TemplateServ
   const server = superstatic({
     debug: false,
     port: port,
-    host: options.host,
+    hostname: options.host,
     config: config,
     compression: true,
-    cwd: detectProjectRoot(options),
+    cwd: detectProjectRoot(options) || undefined,
```

Comment:
Little confused why this `|| undefined` is needed - is this meant to cast falsey values to undefined?

Reply:
Yeah, detectProjectRoot returns `string | null`, which is just irritating. This is why I don't use `null`.